e2e/stack_test.go

```diff
@@ -433,50 +365,104 @@ func ethRollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	require.True(t, finalizeWithdrawalLogs.Event.Success, "withdrawal finalization failed")
 	require.NoError(t, finalizeWithdrawalLogs.Close())
 
-	// get the user's balance after the withdrawal has been finalized
-	balanceAfterFinalization, err := stack.L1Client.BalanceAt(stack.Ctx, userETHAddress, nil)
```

Comment:
Should these balance checks to ensure that the L1 recipient address got the withdrawn funds be added back in?

Reply:
Those only check that the gas for the L1 transaction was deducted, correct? My understanding is that they effectively test the L1 full node, not Monomer.